.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 2.1
```

Comment:
Hmm. Well, chef uses 2.1, not 2.2 (yet). But our ruby should be 2.2 safe, so I'd say yes, go ahead.